src/app/manager-dashboard/reports/reports-detail.component.ts

```diff
@@ -5,6 +5,7 @@ import { ReportsService } from './reports.service';
 import { StateService } from '../../shared/state.service';
 import { Chart } from 'chart.js';
 import { styleVariables, dedupeShelfReduce } from '../../shared/utils';
+import {DialogsLoadingService} from "../../shared/dialogs/dialogs-loading.service";

```

Comment:
@BhargaviNadendla Please make sure you have properly added the git pre push hook to your local repository by running `npm run install-hooks` before pushing to GitHub.  The hook will run the linter prior to pushing and reject the changes if there are any lint errors.  If there are errors you can run `npm run v-lint-fix` which will automatically fix what it can and list the errors for you to fix.